Source/WebCore/dom/TrustedHTML.h

```diff
@@ -31,7 +31,7 @@
 
 namespace WebCore {
 
-class TrustedHTML : public ScriptWrappable, public RefCounted<TrustedHTML> {
+class WEBCORE_EXPORT TrustedHTML : public ScriptWrappable, public RefCounted<TrustedHTML> {
```

Comment:
Why is it needed?

Reply:
I believe this was needed for the build to succeed previously. I will just double check if it's still needed.

Reply:
So it's needed else WebKitLegacy doesn't build, I think it's because Document::execCommand is exported.